safe_cli/operators/hw_wallets/trezor_wallet.py

```diff
@@ -49,3 +50,12 @@ def sign_typed_hash(self, domain_hash: bytes, message_hash: bytes) -> bytes:
             self.client, n=address_n, domain_hash=domain_hash, message_hash=message_hash
         )
         return signed.signature
+
+    def get_signed_raw_transaction(self, tx_parameters: TxParams) -> HexStr:
+        """
+
+        :param chain_id:
+        :param tx_parameters:
+        :return: raw transaction signed
+        """
+        raise NotImplementedError
```

Comment:
Issue to implement this was created https://github.com/safe-global/safe-cli/issues/326